### DISCUSSION TITLE
Error 502

### DISCUSSION THREAD

**Discussion by Danii57** [Melledy/Grasscutter#412] [Support]
Hi, so i'm getting an error (502) that when I try to login it doesn't let me, maybe the solution is on the wiki, but I didn't find it so can someone help me?

**Comment by KormiMeiko** [comment]
ERROR 502:The server was acting as a gateway or proxy and received an invalid response from the upstream server. (from Wikipedia)

Maybe your grasscutter is running abnormally, or the agent configuration is wrong? You can try to check whether these two areas are configured correctly

**Comment by 69gg** [comment]
> Hi, so i'm getting an error (502) that when I try to login it doesn't let me, maybe the solution is on the wiki, but I didn't find it so can someone help me?

You can try turning off iPv6 in the Settings>Network and Internet>Advanced Network Settings>Choose your network adapter>Edit more adapter options>Uncheck iPv6/TCP.
Then restart you Grasscutters Server and your game.
![image](https://github.com/user-attachments/assets/e2ce7c6e-6843-4188-8b9f-cf5edbbf7bb7)
@Danii57
